script/product-list.js

```diff
@@ -3,10 +3,13 @@ const productListGrid = document.querySelector(".products-show--grid");
 const productListLine = document.querySelector(".products-show--line");
 const cartList = document.querySelector(".cart-table");
 const paginationBtn = document.querySelectorAll(".page-link");
+const productMenu = document.querySelector(".products-menu");
 
 let buttonDOM = [];
 let cart = [];
 let products = [];
+let menu = [];
+let isFiltered = "";
```

Comment:
```suggestion
let categoryApplied = "";
```
đặt tên theo như này thì phải là boolean chứ sao lại string nhỉ